ert_storage/endpoints/ensembles.py

```diff
@@ -14,5 +13,63 @@ def post_ensemble(
     experiment = db.query(ds.Experiment).get(experiment_id)
     ens = ds.Ensemble(inputs=ens_in.parameters, experiment=experiment)
     db.add(ens)
+
+    if ens_in.update_id:
+        update_obj = db.query(ds.Update).get(ens_in.update_id)
+        update_obj.ensemble_result = ens
+    db.commit()
+
+    return js.EnsembleOut(
+        id=ens.id,
+        children=[child.ensemble_result_id for child in ens.children],
+        parent=ens.parent.ensemble_reference_id if ens.parent else None,
+    )
+
+
+@router.get("/ensembles/{ensemble_id}", response_model=js.EnsembleOut)
+def get_ensemble(*, db: Session = Depends(get_db), ensemble_id: int) -> js.EnsembleOut:
+    ens = db.query(ds.Ensemble).get(ensemble_id)
+
+    return js.EnsembleOut(
+        id=ens.id,
+        children=[child.ensemble_result_id for child in ens.children],
+        parent=ens.parent.ensemble_reference_id if ens.parent else None,
+    )
+
+
+@router.post("/ensembles/{id}/updates", response_model=js.UpdateOut)
+def create_update(
+    *,
+    db: Session = Depends(get_db),
+    id: int,
```

Comment:
Is this one necessary? For me, it seems like this id should contain the ensemble to attach the update to. But from what i can see on line 50 you are getting this from the `UpdateIn` object through `ensemble_reference_id`. Do I misunderstand something? 🤔

Reply:
yes you are right